assets/src/components/mapPage/mapDisplay.tsx

```diff
@@ -361,6 +366,7 @@ const SelectedRouteDataLayers = ({
     (selectedRoutePattern?.shape?.stops || []).map((s) => s.id)
   )
 
+  const { paddingTopLeft } = useContext(MapSafeAreaContext)
```

Comment:
This isn't my favorite, because we have to repeat this like four times, but I don't ... thoroughly hate it? Since the thing that's repeated is just one line.

Reply:
What if you made a hook that wraps the context and the function?
```
const useFixedZoomDrawerOffsetFollowerFn() => {
	const { paddingTopLeft } = useContext(MapSafeAreaContext)
	return fixedZoomDrawerOffsetAutoCenter(paddingTopLeft)
}
```